### ISSUE TITLE
[SPIKE] Integration Tests for Stanford and Cornell DAGs

### ISSUE THREAD

**Issue by jermnelson** [LD4P/ils-middleware#135]
Investigate and recommend a strategy for DAG integration tests.

Possible strategies:
- Use airflow command-line operations to run Dag with a mocked sqlite database and external APIs calls
- Use [Cypress](https://www.cypress.io/) and docker compose that provisions and runs DAGs with mocked external APIS

**Comment by jcoyne** [comment]
Perhaps I'm not understanding your goals here, but why don't we just run an end-to-end test on staging?

**Comment by jermnelson** [comment]
@jcoyne, agreed. Using our Stage environment would be a better for our integration testing.
